polygeist/tools/cgeist/Test/Verification/stream.cu

```diff
@@ -14,11 +14,7 @@ void run(cudaStream_t stream1, int *array, int n) {
 }
 
 // CHECK:   func.func @_Z3runP10cudaStreamPii(%arg0: !llvm.ptr<struct<()>>, %arg1: memref<?xi32>, %arg2: i32) attributes {llvm.linkage = #llvm.linkage<external>} {
-// CHECK-NEXT:     %c10 = arith.constant 10 : index
-// CHECK-NEXT:     %c1 = arith.constant 1 : index
-// CHECK-NEXT:     %c20 = arith.constant 20 : index
-// CHECK-NEXT:     %0 = "polygeist.stream2token"(%arg0) : (!llvm.ptr<struct<()>>) -> !gpu.async.token
-// CHECK-NEXT:     %1 = gpu.launch async [%0] blocks(%arg3, %arg4, %arg5) in (%arg9 = %c10, %arg10 = %c1, %arg11 = %c1) threads(%arg6, %arg7, %arg8) in (%arg12 = %c20, %arg13 = %c1, %arg14 = %c1) {
+// CHECK:          %13 = gpu.launch async [%12] blocks(%arg3, %arg4, %arg5) in (%arg9 = %1, %arg10 = %3, %arg11 = %5) threads(%arg6, %arg7, %arg8) in (%arg12 = %7, %arg13 = %9, %arg14 = %11) {
```

Comment:
can this be `CHECK-NEXT`?

Reply:
No

Reply:
Why is this test case modified? The removed lines turn into a call?

Reply:
Checked. There are 4 calls, and they are some `affine.load` and `affine.store`, after they can inlined, then they can be simplified. to the removed check lines.

Reply:
I think what matters in this test is to check that the GPU kernel is launched via gpu.launch instruction. So I removed some irrelevant lines.